### PR TITLE
GWWC login template updates

### PR DESCRIPTION
**Pull request by mordaroso**
In the login screen of Auth0
* don't mention ea.org account in subtitle
* use GWWC copy and links in footer callouts

Screenshot 2024-01-15 at 15 42 26

Additionally rename the GWWC app to "Giving What We Can" (remove mention of 'Lyra')

### CODE REVIEW

File: templates/login.liquid [hunk 1]
```diff
@@ -23,6 +23,17 @@
       #prompt-logo-center {
         display: none;
       }
+      header p {
+        text-indent: -9999999px;
```
Comment (sarahbaka) [inline templates/login.liquid:27]: What does the `-9999999px;` do exactly, in the updated design?

Comment (sarahbaka) [inline templates/login.liquid:27]: [Query] What does the `-9999999px;` do exactly, in the updated design?

File: templates/login.liquid [hunk 2]
```diff
@@ -102,13 +113,23 @@
       <div class="widget-wrapper">
         {%- auth0:widget -%}
       </div>
+      {% if application.name contains "Giving" %}
+      <div class="secondary-box">
+        By creating an <a href="https://www.givingwhatwecan.org/" target="_blank" rel="noreferrer">Giving What We Can</a> account,
```
Comment (sarahbaka) [inline templates/login.liquid]:
```suggestion
        By creating a <a href="https://www.givingwhatwecan.org/" target="_blank" rel="noreferrer">Giving What We Can</a> account,
```

File: templates/login.liquid
Comment (sarahbaka) [inline templates/login.liquid:1]: [Query] How exactly is the "Log in to EffectiveAltruism.org to continue to ..." copy removed? I see the removal of the application name  'Giving What We Can - Lyra' in the `cli/manifests.ts` file, but not the removal of "Log in to EffectiveAltruism.org to continue to ..." in the PR diffs. Does simply adding a new `header p:after {` section override this copy?

**Review by sarahbaka — CHANGES_REQUESTED** [review]

**Review by sarahbaka — COMMENTED** [review]
@mordaroso Should be good to be deployed soon, just a sanity check question about where/how "Log in to EffectiveAltruism.org to continue to ..." is removed and a minor copy correction 🙂

File: templates/login.liquid (hunk 1 above)
Comment (mordaroso) [reply templates/login.liquid:27]: It is a [common pattern](https://stackoverflow.com/a/22054588) to hide or replace text. We use the same logic already to remove the "Welcome" and show the GWWC logo (see changes on `h1`).

Here in this case it hides the existing text in the `header p` which is "Login to EffectiveAltruism.org to continue to Giving What We Can" and the `header p:after` adds the new text.

File: templates/login.liquid
Comment (mordaroso) [reply templates/login.liquid:1]: Yes exactly. It first hides the existing text (see css for `header p`) and then adds the new content as `header p:after`. It's a bit of a hack but there seems no way that we use conditionals in the "[Universal Login Custom Text](https://manage.auth0.com/dashboard/us/effectivealtruism/custom_text)" 
Screenshot 2024-01-16 at 09 13 42


As we only want to change the text for the GWWC login screen and keep it for the rest (EA Forum, EAG, etc) we use css to replace it.

Comment (sarahbaka) [reply templates/login.liquid:1]: Ahh, I see now - thanks for clarifying!

**Review by sarahbaka — APPROVED** [review]

**Pull request merged**